src/App.tsx

```diff
@@ -1,14 +1,48 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
it's important, add it to the dependencies list:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/54cfa787-bcd7-40bf-a13e-0a45710fde83)